### PR TITLE
fix rpc_client 无法编译的问题

### PR DESCRIPTION
**Pull request by xiaomM**
Change-Id: I31625e13d3b4ea286b5382249ca075f68b104a25

### Issues associated with this PR

Your PR should present related issues you want to solve.

### Sign the CLA
Make sure you have signed the [CLA](https://www.clahub.com/agreements/alipay/sofa-mosn)

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, 
details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases.
And you need to show the ut result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: examples/codes/tcpproxy-sample/rpc_client.go [hunk 1]
```diff
@@ -69,11 +70,12 @@ func buildBoltV1Request(requestID uint64) *sofarpc.BoltRequest {
 
 	headers := map[string]string{"service": "testSofa"} // used for sofa routing
 
-	if headerBytes, err := serialize.Instance.Serialize(headers); err != nil {
+	var headerBytes = &buffer.IoBuffer{}
```
Comment (nejisama) [inline examples/codes/tcpproxy-sample/rpc_client.go]: buffer.NewIoBuffer
